src/main/java/br/unb/cic/analysis/model/Statement.java

```diff
@@ -16,7 +18,8 @@
 	public enum Type {
 		SOURCE, 
 		SINK,
-		IN_BETWEEN;
+		IN_BETWEEN,
+		SOURCE_SINK
```

Comment:
Na minha branch de trabalho (ioa), eu precisei criar um tipo para usar quando fosse source e sink. Eu dei o nome de BOTH. 

Qual nomenclatura que para vocês faz mais sentido? @pauloborba e @rafaelmotaalves?
Se for  SOURCE_SINK, quando o merge for feito eu atualizo a minha análise.

Reply:
Tá estranho a nomenclatura de SOURCE_SINK. Pra mim, isso significaria conflito textual... Seria algo realizado pelo integrador?

Reply:
Sim @galilasmb , eu decidi considerar para caso façamos algum teste manual ou com alguma ferramenta de merge não textual.